### PR TITLE
Feature/Gitlab Redirect Contributor console

### PR DESCRIPTION
**Pull request by wanyaland**
- Added flow for merge request that initiates the sign request

Signed-off-by: Harold Wanyama <wanyaland+nickmango@gmail.com>

### CODE REVIEW

File: cla-backend-go/config/ssm.go [hunk 1]
```diff
@@ -78,6 +78,7 @@ func loadSSMConfig(awsSession *session.Session, stage string) Config { //nolint
 		fmt.Sprintf("cla-corporate-base-%s", stage),
 		fmt.Sprintf("cla-corporate-v1-base-%s", stage),
 		fmt.Sprintf("cla-corporate-v2-base-%s", stage),
+		fmt.Sprintf("cla-contributor-v2-base-%s", stage),
```
Comment (dealako) [inline cla-backend-go/config/ssm.go:81]: You will need to set the SSM config value down below in the switch/case statement.  This is missing. As a result, the value will be empty.

File: cla-backend-go/config/config.go [hunk 2]
```diff
@@ -67,6 +67,8 @@ type Config struct {
 	CorporateConsoleV1URL string `json:"corporateConsoleV1URL"`
 	CorporateConsoleV2URL string `json:"corporateConsoleV2URL"`
 
+	CLAContributorv2Base string `json:"cla-contributor-v2-base"`
```
Comment (dealako) [inline cla-backend-go/config/config.go:70]: I have added a terraform entry to set this value. Previously it was not managed by CI/CD. See: https://github.com/LF-Engineering/lfx-easycla-terraform/pull/35. We will want to always consider managing SSM values using terraform moving forward. So, if we encounter SSM values that are not managed, we should take the extra step to add it.

File: cla-backend-go/swagger/cla.v2.yaml [hunk 3]
```diff
@@ -4276,6 +4296,24 @@ parameters:
     pattern: '^[a-fA-F0-9]{8}-?[a-fA-F0-9]{4}-?4[a-fA-F0-9]{3}-?[89ab][a-fA-F0-9]{3}-?[a-fA-F0-9]{12}$' # uuidv4
     minLength: 5
     maxLength: 255
+  path-gitlabOrganizationID:
+    name: organizationID
+    description: gitlab organization ID
```
Comment (dealako) [inline cla-backend-go/swagger/cla.v2.yaml:4301]: Let's make this description consistent with the others, make it say 'GitLab organization ID'.  I am not good with this either, but together can clean up the documentation generated from the swagger build process.

File: cla-backend-go/swagger/cla.v2.yaml [hunk 4]
```diff
@@ -4276,6 +4296,24 @@ parameters:
...
+    type: string
+    in: path
+    required: true
+  path-mergeRequestID:
+    name: mergeRequestID
+    description: Gitlab Merge Request identifier
```
Comment (dealako) [inline cla-backend-go/swagger/cla.v2.yaml:4307]: Change Gitlab to be GitLab.

File: cla-backend-go/swagger/cla.v2.yaml [hunk 5]
```diff
@@ -4276,6 +4296,24 @@ parameters:
...
+    type: string
+    in: path
+    required: true
+  path-gitlabRepositoryID:
+    name: gitlabRepositoryID
+    type: string
+    description: Gitlab Repository/Project identifier
```
Comment (dealako) [inline cla-backend-go/swagger/cla.v2.yaml:4314]: Change Gitlab to be GitLab.

**Review by dealako — CHANGES_REQUESTED** [review]
Minor feedback.

File: cla-backend-go/config/config.go (hunk 2 above)
Comment (nickmango) [reply cla-backend-go/config/config.go:70]: Thanks. Had only checked this via the console

**Pull request merged**